packages/react-devtools-shared/src/devtools/views/Components/EditableValue.js

```diff
@@ -28,6 +28,7 @@ export default function EditableValue({
   value,
 }: EditableValueProps) {
   const inputRef = useRef<HTMLInputElement | null>(null);
+  const checkBoxRef = useRef<HTMLInputElement | any>(null);
```

Comment:
We don't need `checkBoxRef` (or `inputRef`) :)

Reply:
yes i see that now! thanks!